lib/lang/vi.js

```diff
@@ -0,0 +1,167 @@
+  "very-light-precipitation": "mưa nhỏ",
+  "possible-light-precipitation": "có thể có mưa nhẹ",
+  "light-precipitation": "lượng mưa nhỏ",
+  "medium-precipitation": "lượng mưa trung bình",
+  "heavy-precipitation": "lượng mưa lớn",
+  "possible-very-light-rain": "có thể có mưa phùn",
+  "very-light-rain": "mưa phùn",
+  "possible-light-rain": "có thể có mưa nhỏ",
+  "light-rain": "mưa nhỏ",
+  "medium-rain": "mưa vừa",
+  "heavy-rain": "mưa to",
+  "possible-very-light-sleet": "có thể tuyết rơi nhỏ",
+  "very-light-sleet": "mưa tuyết nhẹ",
+  "possible-light-sleet": "có thể có mưa tuyết nhỏ",
+  "light-sleet": "mưa tuyết nhỏ",
+  "medium-sleet": "mưa tuyết vừa",
+  "heavy-sleet": "mưa tuyết to",
+  "possible-very-light-snow": "có thể có tuyết rơi nhỏ",
+  "very-light-snow": "tuyết rơi rất nhỏ",
+  "possible-light-snow": "có thể có tuyết rơi nhỏ",
+  "light-snow": "tuyết rơi nhỏ",
+  "medium-snow": "tuyết rơi",
+  "heavy-snow": "tuyết rơi nhiều",
+  "possible-thunderstorm": "có thể có dông",
+  "thunderstorm": "có dông",
+  "light-wind": "gió nhẹ",
+  "medium-wind": "có gió",
+  "heavy-wind": "gió to",
+  "low-humidity": "trời hanh khô",
+  "high-humidity": "trời nồm",
+  "fog": "có sương mù",
+  "light-clouds": "ít mây",
+  "medium-clouds": "có mây",
+  "heavy-clouds": "trời âm u",
+  "today-morning": "sáng nay",
+  "later-today-morning": "cuối buổi sáng",
+  "today-afternoon": "trưa nay",
+  "later-today-afternoon": "chiều hôm nay",
+  "today-evening": "chiều tối nay",
+  "later-today-evening": "tối hôm nay",
+  "today-night": "đêm nay",
+  "later-today-night": "nửa đêm",
+  "tomorrow-morning": "sáng mai",
+  "tomorrow-afternoon": "trưa mai",
+  "tomorrow-evening": "chiều tối mai",
+  "tomorrow-night": "tối mai",
+  "morning": "vào buổi sáng",
+  "afternoon": "vào buổi chiều",
+  "evening": "vào buổi tối",
+  "night": "đêm",
+  "today": "hôm nay",
+  "tomorrow": "vào ngày mai",
+  "sunday": "vào chủ nhật",
+  "monday": "vào thứ hai",
+  "tuesday": "vào thứ ba",
+  "wednesday": "vào thứ tư",
+  "thursday": "vào thứ năm",
+  "friday": "vào thứ sáu",
+  "saturday": "vào thứ bảy",
+  "next-sunday": "chủ nhật tuần sau",
+  "next-monday": "thứ hai tuần sau",
+  "next-tuesday": "thứ ba tuần sau",
+  "next-wednesday": "thứ tư tuần sau",
+  "next-thursday": "thứ năm tuần sau",
+  "next-friday": "thứ sáu tuần sau",
+  "next-saturday": "thứ bảy tuần sau",
+  "minutes": "$1 phút",
+  "fahrenheit": "$1\u00B0F",
+  "celsius": "$1\u00B0C",
+  "inches": "$1 in",
+  "centimeters": "$1 cm",
+  "less-than": "dưới $1",
+  "and": function(a, b) {
+    return join_with_shared_prefix(
+      a,
+      b,
+      a.indexOf(",") !== -1 ? ", và " : " và "
+    );
+  },
+  "through": function(a, b) {
+    return join_with_shared_prefix(a, b, " cho tới ");
+  },
+  "with": "$1, với $2",
+  "range": "$1\u2013$2",
+  "parenthetical": function(a, b) {
+    return a + " (" + b + (a === "mixed precipitation" ? " of snow)" : ")");
+  },
```

Comment:
`"of snow"` is not translated here! Either it should be, or else this should simply be `"$1 ($2)"`.

Reply:
I will fix it